app/assets/stylesheets/application.scss

```diff
@@ -69,4 +73,22 @@ nav {
   margin-top: 5vh;
 }
 
+.big-logo {
+  width: 70vh;
+}
+
+.bw {
+  padding-top: 7vh;
```

Comment:
Properties should be ordered -webkit-filter, padding-top, -moz-transition, -ms-transition, -o-transition, -webkit-transition, transition